addon/components/links-with-follower.js

```diff
@@ -62,7 +62,7 @@ export default Ember.Component.extend({
    * @type {Number}
    * @default 150
    */
-  followerAnimationDuration: 150,
+  followerAnimationDuration: (Ember.testing) ? 0 : 150,
```

Comment:
Cool! Taken care of in c07dcec. Would love to get your 👀 on that to see if that technique makes sense.